Pipfile

```diff
@@ -13,6 +13,8 @@ pytest-mock = "*"
 docker = "*"
 docker-squash = "*"
 click = "*"
+pylint = "<2.0.0"
```

Comment:
Why < version 2 ? It seems F29 has 2.1.1 ...

Reply:
I missed it, it was automatically added by IDE.